service/entityresolution/entity_resolution.proto

```diff
@@ -26,12 +23,12 @@ message IdpConfig {
    }
 
 */
-message IdpPluginRequest {
+message ResolveEntitiesRequest {
     repeated authorization.Entity entities = 1;
 }
 
 
-message IdpEntityRepresentation {
+message ERSEntityRepresentation {
```

Comment:
Do we need to prefix this with ERS?

Reply:
removed, there used to be another EntityRepresentaion object and i wanted to make sure they were distinguishable